python/tvm/auto_scheduler/measure.py

```diff
@@ -889,18 +929,43 @@ def _timed_rpc_run(
 
     if error_no == 0:
         try:
-            args = [ndarray.empty(get_const_tuple(x.shape), x.dtype, ctx) for x in build_res.args]
-            try:
-                random_fill = remote.get_function("tvm.contrib.random.random_fill")
-            except AttributeError:
-                raise AttributeError(
-                    "Please make sure USE_RANDOM is ON in the config.cmake " "on the remote devices"
-                )
-            for arg in args:
-                random_fill(arg)
+            if os.path.exists(working_dir):
+                buffer_path = os.path.join(working_dir, "buffer.pkl")
+                if os.path.exists(buffer_path):
+                    with open(buffer_path, "rb") as fi:
+                        buffer = pickle.load(fi)
+                    # force last args to be empty
+                    args = []
+                    for i in range(len(build_res.args) - 1):
+                        args.append(ndarray.array(buffer[build_res.args[i].name], ctx=ctx))
```

Comment:
This will cause a copy over RPC for every measurement. A better way is to copy the `buffer.pkl` to the remote device once and do the initialization/check on the remote device.
We can leave a "todo" here and leave this optimization to future PRs.